aksel.nav.no/website/pages/eksempler/bodyshort/colors.tsx

```diff
@@ -2,31 +2,21 @@ import { BodyShort, VStack } from "@navikt/ds-react";
 import { withDsExample } from "@/web/examples/withDsExample";
 
 const Example = () => {
-  const lorem = "Du må gjøre en filtrering for å se brukere i listen.";
+  const exampleText = "Du må gjøre en filtrering for å se brukere i listen.";
```

Comment:
Er dette et godt eksempel på tekst for BodyShort? 🤔

Reply:
Tenker den fungerer helt ok (finner ikke på noen bedre... 🙈 )